### PR TITLE
Remove redundant passing of language

### PR DESCRIPTION
**Pull request by mbrdl**
Language is contained in the options, passing it separately is entirely pointless

### CODE REVIEW

File: core/src/main/java/de/jplag/SubmissionSetBuilder.java [hunk 1]
```diff
@@ -31,19 +31,14 @@ public class SubmissionSetBuilder {
...
     /**
      * Creates a builder for submission sets.
-     * @param language is the language of the submissions.
      * @param options are the configured options.
      */
-    public SubmissionSetBuilder(Language language, JPlagOptions options) {
-        this.language = language;
+    public SubmissionSetBuilder(JPlagOptions options) {
         this.options = options;
-        this.excludedFileNames = options.excludedFiles();
```
Comment (dfuchss) [inline core/src/main/java/de/jplag/SubmissionSetBuilder.java:46]: JPlag has to ensure backward compatibility. Therefore, you could mark the old constructor as deprecated and just pass the option to a new constructor. But we cannot remove the constructor directly.

**Review by dfuchss — CHANGES_REQUESTED** [review]
In general. the PR looks good to me. One remaining issue needs to be addressed.

File: core/src/main/java/de/jplag/SubmissionSetBuilder.java (hunk 1 above)
Comment (mbrdl) [reply core/src/main/java/de/jplag/SubmissionSetBuilder.java:46]: Is the SubmissionSetBuilder part of the interface? I though it was entirely internal, thus making backwards compatability unnecessary

Comment (dfuchss) [reply core/src/main/java/de/jplag/SubmissionSetBuilder.java:46]: It's a public class in the core module. Therefore, I think we have to consider it as part of the interface

**Comment by mbrdl** [comment]
I'll just make this PR part of #1041

**Pull request closed without merging**